unitTests/utils/DomTest.ts

```diff
@@ -34,17 +34,17 @@ export function DomTests() {
       describe('when calling #isValidElement', () => {
         it('should respond true when a valid element is given', () => {
           const NORMAL_ELEMENT = document.createElement('div');
-          expect($$(NORMAL_ELEMENT).isValid()).toBeTruthy;
+          expect($$(NORMAL_ELEMENT).isValid()).toBeTruthy();
         });
```

Comment:
I found these tests on the master branch that were not asserting anything 😄

Reply:
toBeTruthy() or not toBeTruthy, that is the question